test/driver/mysql_test.js

```diff
@@ -119,7 +125,7 @@ driver.connect(config, internals, function(err, db) {
           assert.equal(column.getDataType(), 'TINYINT');
           assert.equal(column.isNullable(), true);
           assert.equal(column.getDefaultValue(), 0);
-        }
+        },
```

Comment:
Then push the change to your branch ;)

I will take a look at this in about two hours, currently still busy.